source/extensions/matching/http/cel_input/cel_input.h

```diff
@@ -43,7 +43,7 @@ class HttpCelDataInput : public Matcher::DataInput<Envoy::Http::HttpMatchingData
     // and attributes from stream info.
     std::unique_ptr<google::api::expr::runtime::BaseActivation> activation =
         Extensions::Filters::Common::Expr::createActivation(
-            data.streamInfo(), maybe_request_headers.ptr(), maybe_response_headers.ptr(),
+            nullptr, data.streamInfo(), maybe_request_headers.ptr(), maybe_response_headers.ptr(),
```

Comment:
Please put a TODO on all the nullptr local info places with an issue reference so we can fix them later.